src/leap/mail/mail.py

```diff
@@ -190,11 +212,15 @@ def get_subpart(self, part):
 
         sub_pmap = self._pmap.get("part_map", {})
         try:
-            part_map = sub_pmap[str(part + 1)]
+            part_map = sub_pmap[str(part)]
         except KeyError:
-            logger.debug("getSubpart for %s: KeyError" % (part,))
+            log.msg("getSubpart for %s: KeyError" % (part,))
```

Comment:
Then we'll have to see how to integrate them nicely in our logging env as now they look like:
```
[2015-07-08 18:44:15] INFO     - L#109  : twisted.logger._stdlib:__call__ - getSubpart for 9: KeyError
```

While the other logs looks like:
```
[2015-07-08 18:48:10] DEBUG    - L#103  : leap.soledad.client.sync:sync - Soledad sync: there are 0 documents to send.
```

We are loosing information about from where is this log coming, it's severity, ...